README.md

```diff
@@ -113,15 +113,15 @@ role_session_name = example
 The [Packer template](src/packer.json) requires two environment
 variables to be defined:
 
-* `BUILD_REGION`: The region in which to build the image.
-* `BUILD_REGION_KMS`: The KMS key alias to use to encrypt the image.
+- `BUILD_REGION`: The region in which to build the image.
```

Comment:
We should also mention the `BUILD_BUCKET` env variable here, similar to what we did over in [assessor-portal-packer](https://github.com/cisagov/assessor-portal-packer/pull/1#pullrequestreview-711944089).  While it is not strictly required (since there is a default value), I think it's fine to just say that it is required here.

Reply:
See commit e0a1a33.

Reply:
You also need to change the line above it from "two" to "three" environment variables.

Reply:
See commit 4c3f3eb.

Reply:
👍